### DISCUSSION TITLE
Being able to change the color of the UI/changing the color theme of your game

### DISCUSSION THREAD

**Discussion by Modsnotworking** [ppy/osu#15238] [Ideas] [answered]
I have been wondering lately after the release of the "Settings"UI redesign which has a purpulish color. What if there's a possibility to change the color of UI
For Example: Being able to change the color of setting UI from the purpulish color to any other color, like, to pink or turqouise, also the option to change color of the top bar UI, i.e, the rulesets, changelog, wiki, user button(Changing with the help of the color proivder used while editing the colors of circles in the editor?)

**Comment by peppy** [comment, marked as answer]
We eventually intend for an accent colour to be customisable, but this will probably not cover the actual game colour schemes. Which match the web and are intentionally locked for recognition.

So no, probably not to the level you want.

**Reply by Modsnotworking** [reply]
> We eventually intend for an accent colour to be customisable, but this will probably not cover the actual game colour schemes. Which match the web and are intentionally locked for recognition.
> 
> So no, probably not to the level you want.

This will mean that if the color scheme used in the web change at some point in the future then the colors used in the UI in lazer will change too to match the color scheme in web that got changed. just to summarized, when the web colors change then eventually the colors in the UI of lazer will change too to match the colors used in the web? BUT we won't be able to change the color of the UI in lazer in our own???

Well if what I said is true then marking your reply as an answer will make sense

**Reply by peppy** [reply]
i dont know man.
